lib/licensee/matchers/cabal.rb

```diff
@@ -4,12 +4,29 @@ class Cabal < Licensee::Matchers::Package
       # While we could parse the cabal file, prefer
       # a lenient regex for speed and security. Moar parsing moar problems.
       LICENSE_REGEX = /^\s*license\s*\:\s*([a-z\-0-9\.]+)\s*$/ix.freeze
+      LICENSE_CONVERSIONS = {
+        'GPL-2'      => 'GPL-2.0',
+        'GPL-3'      => 'GPL-3.0',
+        'LGPL-2.1'   => 'LGPL-2.1',
+        'LGPL-3'     => 'LGPL-3.0',
+        'AGPL-3'     => 'AGPL-3.0',
+        'BSD2'       => 'BSD-2-Clause',
+        'BSD3'       => 'BSD-3-Clause',
+        'MIT'        => 'MIT',
+        'ISC'        => 'ISC',
+        'MPL-2.0'    => 'MPL-2.0',
+        'Apache-2.0' => 'Apache-2.0'
```

Comment:
I don't think the last 4 are needed as they are no-ops.

Reply:
Ooops, you are right :ok_hand:. Removing them and `LGPL-2.1` which is already in standard SPDX format.